src/details/ArborX_DetailsDistributedTreeImpl.hpp

```diff
@@ -42,6 +42,34 @@ struct DefaultCallbackWithRank
   }
 };
 
+KOKKOS_INLINE_FUNCTION Sphere buildSphere(Point const &center, float radius)
+{
+  return Sphere{center, radius};
+}
+
+KOKKOS_INLINE_FUNCTION Sphere buildSphere(Box const &box, float distance)
+{
+  auto const min_x = box.minCorner()[0];
+  auto const min_y = box.minCorner()[1];
+  auto const min_z = box.minCorner()[2];
+  auto const max_x = box.maxCorner()[0];
+  auto const max_y = box.maxCorner()[1];
+  auto const max_z = box.maxCorner()[2];
+  Point center{0.5f * (min_x + max_x), 0.5f * (min_y + max_y),
+               0.5f * (min_z + max_z)};
+  float radius =
+      distance + Kokkos::Experimental::sqrt((max_x - min_x) * (max_x - min_x) +
+                                            (max_y - min_y) * (max_y - min_y) +
+                                            (max_z - min_z) * (max_z - min_z));
```

Comment:
Please define `KokkosExt::hypot` in `ArborX_DetailsKokkosExtMathFunctions.hpp` and report on the Kokkos side we want the ternary function overload.